OpenRA.Game/Server/Server.cs

```diff
@@ -376,7 +376,7 @@ void ValidateClient(Connection newConn, string data)
 				Log.Write("server", "{0} ({1}) has joined the game.",
 					client.Name, newConn.Socket.RemoteEndPoint);
 
-				if (LobbyInfo.NonBotClients.Count() > 1)
+				if (LobbyInfo.NonBotClients.Count() >= 1)
```

Comment:
This doesn't look right.  `SendMessage` should already do the right thing when there are no clients, so if this check really is for "more than zero players" then it can be removed completely.

I suspect that the original intention may have been to send a message to all *other* players?  If this is the case, then the real issue is that it prints this for the joining player whenever there are other clients on the server.

Reply:
Hi @pchote,

To my understanding the mentioned issue was about missing message in console for the first player:
> When a new instance of a dedicated server starts, and the first player joins, the first player's name and announcement "[TIMESTAMP] PlayerNameHere has joined the game." does not display in the console output.
This check of `NonBotClients.Count()` is messing it up, resulting in message displaying only when there is at least one player.

And this is an entirely different issue:
> I suspect that the original intention was to send a message to all other players, and never the newly joined client? If this is the case, then the real issue is that it prints this for the joining player whenever there are other clients on the server.
If this was an intended feature I could also work on this change.

Reply:
The issue is that `LobbyInfo.NonBotClients.Count() >= 1` is by definition always true, so the if statement becomes completely redundant.